### PR TITLE
Update device-control-report.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated broken link

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5947

File: microsoft-365/security/defender-endpoint/device-control-report.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ ms.technology: mde
 ---
 # Protect your organization's data with device control
 
-**Applies to:** [Microsoft Defender for Endpoint](https://go.microsoft.com/fwlink/p/?linkid=2069559)
+**Applies to:** - [Microsoft Defender for Endpoint](https://go.microsoft.com/fwlink/p/?linkid=2146806)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-report.md]:
```suggestion
**Applies to:** [Microsoft Defender for Endpoint](https://go.microsoft.com/fwlink/p/?linkid=2146806)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, minor edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**